_content/language-and-grammar.md

```diff
@@ -1,190 +1,388 @@
 
-| Recommended                                          | Not recommended                           |
-| :--------------------------------------------------- | :---------------------------------------- |
-| **Download the Agora SDK. (Imperative Mood)**        | The Agora SDK can be downloaded.          |
-| **You can download the Agora SDK. (Direct Address)** | The developer can download the Agora SDK. |
+| Do this...          | ...instead of this      |
+| :------------------- | :------------------- |
+| v3.0.0 was released on Mar 4, 2020. | v3.0.0 is released on Mar 4, 2020. |
+| Disconnecting the microphone caused the audio to freeze. | Disconnecting the microphone causes the audio to freeze. |
+
+Avoid phrasing using *will* and *can*. //TODO William Why are you singling these out? "Will" is attached to the future tense, and "can" is conditional--perhaps there are occasions when it applies?
+
+Use the future tense only if there is a significant time delay that matters in context. For example, "Agora will grant you access to the service once your application has been approved." //TODO William I'm not sure about this example. Is there a better one?
+
+The present prefect tense is used to emphasize a past event that has consequences that continue into the present. For example, "Get the information from the log file if error logging has been activated." The corresponding Chinese is 已经. //TODO William The Chinese translates to "already"--is this useful here?
+
+### Possessives
+
+**Quick reference:**
+- Do not use possessives with company or product names.
+_________
 
-### Gender-neutral
+For singular nouns, add "'s" to the end of the word to form a possessive.
+For plural nouns that end in "s", like most English words, add an apostrophe on the end without the "s".
+For plural nouns that do not end in "s", add "'s" to the end of the word. 
 
-- Avoid using "his" or "her", or "he/she".
-- Use the second person, "you", or the collective noun.
+``` admonition:: Example
+
+   Combine the teacher's video with each of the students' videos on the children's channel.
+```
+
+Do not use a possessive with a product or company name. This is due to matters of trademark, because the "'s" may be mistaken for a part of the name, and because trademarks are often considered adjectives rather than nouns. //TODO William Maybe update? The Google Style Guide allows 's with a company name (but not with products). I am not sure why this distinction is made--perhaps they consider a company to be an entity that can "have things". I do not mind the Google rule, because something like "Agora's livestreaming products" or "Apple's latest iOS update" sound okay to me, and sometimes writing around the possessive to avoid it can sound clunky. However, I am okay with either rule, so you should just decide and let me know.
```

Comment:
We have discussed this and decided to keep the current rule.

Reply:
Okay!